Form.css

```diff
@@ -0,0 +1,24 @@
+.Form {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
```

Comment:
This is likely a mistake. Because these are `50%` width and utilizing `display: flex`, they are probably intended to be a displayed in a two-column layout. Rather confusingly, `flex-direction: column' means that the elements in the container will form a column (stacked) rather than meaning that the elements in the container will be arranged in columns (side-by-side).